Makefile.am

```
@@ -161,6 +161,10 @@ baseline_filtered.info: baseline.info
                       "$(abs_builddir)/depends/x86_64-unknown-linux-gnu/include/boost/*" \
                       "$(abs_builddir)/depends/x86_64-unknown-linux-gnu/include/gmock/*" \
                       "$(abs_builddir)/depends/x86_64-unknown-linux-gnu/include/gtest/*" \
+                      "$(abs_builddir)/depends/x86_64-linux-gnu/include/*.h" \
```

I'd suggest the following to get rid of some duplication:

```suggestion
                      "$(abs_builddir)/depends/$(host_triplet)/include/*.h" \
```

---

Thanks, I saw it has been defined along with `build_triplet`, but it is never used elsewhere. Also, I would like tokeep the same `lcov` filters that have been defined in `rpc-tests.sh`.